app/assets/javascripts/criteria.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
I was not sure whether to keep them or not. Also rails generated this file with a .js.coffee extension. Are we going to move to using this type of file later on?
I will delete the files I am not using.